wireguard/latest/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM to-be-replaced-by-local-ref/base:ubuntu
```

Comment:
don't feel bound to the scylla-operator prod runtime image for this test image, especially given the hacks needed to install it. Would this be easier on a different platform, like with
```
podman run -it --rm fedora:38 dnf install -y wireguard-tools wgctrl
```